examples/get-started/flutter-for/ios_devs/lib/navigation.dart

```diff
@@ -0,0 +1,121 @@
+import 'package:flutter/cupertino.dart';
+import 'package:flutter/material.dart';
+
+void main() {
+  runApp(
+    // create an instance of our app
+    // and passes it to the runApp function
+    const App(),
+  );
+}
+
+// #docregion Routes
+// defines the route name as a constant
+// so that it's reusable
+const detailsPageRouteName = '/details';
+
+class App extends StatelessWidget {
+  const App({
+    super.key,
+  });
+
+  @override
+  Widget build(BuildContext context) {
+    return CupertinoApp(
+      home: const HomePage(),
+      // The routes property defines the available named routes
+      // and the widgets to build when navigating to those routes
+      routes: {
+        detailsPageRouteName: (context) => const DetailsPage(),
+      },
+    );
+  }
+}
+// #enddocregion Routes
+
+// create a class that holds each person's data
+@immutable
+class Person {
+  final String name;
+  final int age;
+  const Person({
+    required this.name,
+    required this.age,
+  });
+}
+
+// then create a list of people
+final mockPersons = Iterable.generate(
+  100,
+  (index) => Person(
+    name: 'Person #${index + 1}',
+    age: 10 + index,
+  ),
+);
+
+// This stateless widget displays the list of persons
+// that we get from the mockPersons list and allows the user
+// to tap each person to see their details
+class HomePage extends StatelessWidget {
+  const HomePage({super.key});
+
+  @override
+  Widget build(BuildContext context) {
+    return CupertinoPageScaffold(
+      navigationBar: const CupertinoNavigationBar(
+        middle: Text(
+          'Pick a person',
+        ),
+      ),
+      child: Material(
+        child:
+            // #docregion ListView
+            ListView.builder(
+          itemCount: mockPersons.length,
+          itemBuilder: (context, index) {
+            final person = mockPersons.elementAt(index);
+            final age = '${person.age} years old';
+            return ListTile(
+              title: Text(person.name),
+              subtitle: Text(age),
+              trailing: const Icon(
+                Icons.arrow_forward_ios,
+              ),
+              onTap: () {
+                // when a ListTile that represents a person is
```

Comment:
Similar comment for doc formatting - Capitals, periods, [brackets] around variables/properties.